ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -56,7 +56,7 @@
   private static final String REQUEST_BODY_KEY_URI = "uri";
   private static final String REQUEST_BODY_KEY_FORMDATA = "formData";
   private static final String USER_AGENT_HEADER_NAME = "user-agent";
-
+  private static final int CHUNK_TIMEOUT_NS = 100 * 1000000; // 100ms
```

Comment:
@satya164 This is the old code, but in previous version, this code was deleted. Before that version, this code is used in downloading scenario. I just reuse this code. Thus I don't know what ```NS``` really mean